blocks/footer/footer.js

```diff
@@ -1,21 +1,155 @@
-import { getMetadata } from '../../scripts/aem.js';
-import { loadFragment } from '../fragment/fragment.js';
+import {
+  decorateSections,
+  updateSectionsStatus,
+} from '../../scripts/aem.js';
+
+function onClick() {
+  const disclaimerDiv = document.querySelector('.panel-body');
+  const iconDiv = document.querySelector('.toggle-icon');
+  if (disclaimerDiv.classList.contains('slide-up')) {
+    // If the content is currently hidden, slide it down
+    disclaimerDiv.classList.remove('slide-up');
+    iconDiv.textContent = '-';
+  } else {
+    // If the content is currently visible, slide it up
+    disclaimerDiv.classList.add('slide-up');
+    iconDiv.textContent = '+';
+  }
+}
+
+function decorateFooterTop(block) {
+  const footerTop = block.querySelector('.footer-top');
+  const defaultWrapper = footerTop.querySelector('.default-content-wrapper');
+  const children = [...footerTop.querySelector('.default-content-wrapper').children];
+  defaultWrapper.innerHTML = '';
+  const rowElement = document.createElement('div');
+  defaultWrapper.appendChild(rowElement);
+
+  const col1 = document.createElement('div');
+  col1.classList.add('col-1');
+  col1.appendChild(children[0]);
+  const imageElement = document.createElement('img');
+  const anchorDiv = children[1].childNodes[0];
+  imageElement.src = anchorDiv.href;
```

Comment:
We should directly place the image in the word doc instead of link to the logo.

Reply:
agreed. working on it